multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -10,7 +10,7 @@
 @automm_presets.register()
 def default():
     return {
-        "model.names": ["categorical_mlp", "numerical_mlp", "timm_image", "hf_text", "clip", "fusion_mlp"],
+        "model.names": ["categorical_mlp", "numerical_mlp", "timm_image", "hf_text", "fusion_mlp"],
```

Comment:
wondering if it is intended to remove `clip` from all presets?

Reply:
This is about removing clip in our default presets. We need to have a separate PR for improving our presets logic.